### PR TITLE
F16/my songs

### PR DESCRIPTION
**Pull request by imalovelyghost**
#16 

Song Router
Song Model
Song Schema
Song Seeder
Song Controller
-Get all Songs
-Get song by ID
-Update song by ID

### CODE REVIEW

File: server/src/routes/songRoutes.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const Router = require("express").Router;
+// const { authMiddleware } = require("../middleware");
+const { songController } = require("../controllers");
+
+const songRouter = Router();
+
+//http://localhost:4000/songs/
+songRouter.get("/all", songController.fetchSongs);
+songRouter.get("/:id", songController.getSongById);
+songRouter.patch("/:id", songController.updateSong);
```
Comment (JordixArnau) [inline server/src/routes/songRoutes.js:10]: Please, insert the authMiddleware

File: server/src/db/seed.js [hunk 2]
```diff
@@ -22,15 +23,85 @@ async function seedUsers() {
...
       firebase_id: "I3wKx6kfyXZykQ77pAAMQK3vQ7s1",
     }).lean();
+
     if (!userExists) {
       await mongoose.connection.collection("users").drop();
       seedUsers();
     }
+  } catch (err) {
+    console.log(err);
+    console.log("user errorr");
```
Comment (JordixArnau) [inline server/src/db/seed.js:92]: Watch out for unnecessary console logs ;)

**Review by JordixArnau — CHANGES_REQUESTED** [review]
Please, look at the comment to add the authMiddleware.

File: server/src/routes/songRoutes.js (hunk 1 above)
Comment (imalovelyghost) [reply server/src/routes/songRoutes.js:10]: On my way to fix this! Thank you for the review!

File: server/src/db/seed.js (hunk 2 above)
Comment (imalovelyghost) [reply server/src/db/seed.js:92]: On my way to fix this! Thank you for the review!

**Review by JordixArnau — APPROVED** [review]
Good job!

**Review by solaz0824 — APPROVED** [review]
🙂👌

**Pull request merged**